uw-frame-components/css/themes/uw-stevens-point-variables.less

```diff
@@ -0,0 +1,16 @@
+/* UW Stevens Point colors */
+@color1: #512698; 
+@color2: lighten(@color1, 5%);
+@color3: darken(@color1, 5%);
+@color4: #E1DCCC;
```

Comment:
Only color1 was specified in the design subtask. Lets go with this and see what it looks like. I'm not sure how color4 is used.